src/style.css

```diff
@@ -0,0 +1,129 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+@font-face {
+  font-family: AvenirBold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 20px 8px;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.first_search {
```

Comment:
if I wanna more search do I need to add new class n_search?

let's make this name better